Code/C++/remove_duplicates.cpp

```diff
@@ -0,0 +1,107 @@
+#include <bits/stdc++.h>
+using namespace std;
+
+/*
+
+ Problem : Remove duplicates from an unsorted linked list
+ Algorithm : This problem can be solved by hashing. The elements are hashed to a hash array and elements with frequency more than 1 are picked only once.
+ Time Complexity : O(n)
```

Comment:
Provide the time and space complexity at the end of the code(after the test cases), and also mention that what "n" stands for.

Reply:
@Rohan27s Check now